.github/workflows/automation-scripts/update-changelogs.mjs

```diff
@@ -104,6 +104,10 @@ await $`find ./temp-target-changelogs/ -type f -name "*CHANGELOG.json" -exec sh
 // delete temps
 await $`rm -r ${targetPath}`;
 await $`rm -r ${incomingPath}`;
+// copy {release-version}.md to target branch if the commit that triggered this script run is from a major or minor version bump
+if (commitMessage.endsWith(".0")) {
```

Comment:
on master, nextver.md will be reset right?

Reply:
The nextversion.md clear/reset is handled by https://github.com/iTwin/itwinjs-core/blob/master/common/config/azure-pipelines/jobs/version-bump.yaml#L240C7-L242C60 in the pipeline. Only when the pipeline fails, the clearing is not done. 

Past commits that did this job successfully when pipeline ran correctly presented as evidence:
- https://github.com/iTwin/itwinjs-core/commit/5d994c4a65ada16abb99b1cc7bf98c6fb8006a9b#diff-4a7785af3b56bc69d2ad9f34257cd4a96cba53855aa039abeb761613d0f81988
- https://github.com/iTwin/itwinjs-core/commit/a0d9715300db014546c9bbe22512865a0e8d9e5a#diff-4a7785af3b56bc69d2ad9f34257cd4a96cba53855aa039abeb761613d0f81988
- https://github.com/iTwin/itwinjs-core/commit/333cf6b555c55ff09de23ff9fed16c1e724bace6#diff-4a7785af3b56bc69d2ad9f34257cd4a96cba53855aa039abeb761613d0f81988